vizro-core/docs/pages/user-guides/container.md

```diff
@@ -157,4 +157,75 @@ To create nested containers, add a `Container` to the `components` argument of a
 vm.Container(title="Parent Container", components=[vm.Container(title="Child Container", components=[vm.Button()])])
```

Comment:
```suggestion
vm.Container(
    title="Parent Container",
    components=[
        vm.Container(
            title="Child Container",
            components=[vm.Button()],
        )
    ]
)
```

This in one line at the moment makes it way too hard to understand the structure of the nested contains.